astromodels/tests/test_template_model.py

```diff
@@ -86,6 +92,8 @@ def test_template_factory():
 
     tm(energies)
 
+    tm.clean()
+    
 
 
 
```

Comment:
Maybe add one more test to test the conversion from old table model to new one? Without that future changes in the table model may break this conversion without us noticing.

Reply:
done!